dev/tasks/python-wheels/travis.osx.yml

```diff
@@ -41,7 +41,7 @@ before_install:
   - brew uninstall boost cgal postgis sfcgal
   - brew update
   - brew upgrade cmake
-  - brew install bison flex
+  - brew install bison flex llvm@7
```

Comment:
Does this link to the static libs then? We should not depend on `brew` packages being installed.

Reply:
AFAIK We link llvm statically in all cases. I'll remove llvm in the test section to check this.